VideoWeb/VideoWeb.AcceptanceTests/Steps/JudgePrivateConsultationSteps.cs

```diff
@@ -85,8 +85,9 @@ public void WhenTheyEnterPrivateConsultationRoom()
         {
             _browsers[_c.CurrentUser].Driver.WaitUntilVisible(JudgeWaitingRoomPage.HearingTitle).Displayed.Should().BeTrue();
             WaitForUserStatusToBe(ParticipantState.Available);
+            Thread.Sleep(1500);  // unfortunately need a hard sleep as the button is active in html but does not respond for another second.
```

Comment:
Why not use the `WaitUntilVisible` method instead of sleeping?

Reply:
WaitUntilVisible did not work, the button **was** visible and also active, however there's a background delay before a click on the button will actually work.

Reply:
@MichaelKeaneathmcts it's a 🚨 when thread sleep is dropped across the code. Also when someone enters a consultation room, why not wait for the status to be updated to "InConsultation" and then you know they're actually in the consultation room